apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -134,6 +138,7 @@ registerPlugin( 'page-templates', {
 	render: function() {
 		return (
 			<PageTemplatesPlugin
+				initiallyOpened={ templates.length > 0 }
```

Comment:
What's the benefit of using an additional prop over setting `isOpen` based on `templates.length`?

Reply:
it's converted into an object for the component (not an array anymore) so we can either do `keys( templates ).length > 0` or use a new prop